content/en/docs/reference/using-api/cel.md

```diff
@@ -61,13 +61,38 @@ Example CEL expressions:
 | `self.names.size() == self.details.size() && self.names.all(n, n in self.details)` | Validate the 'details' map is keyed by the items in the 'names' listSet           |
 {{< /table >}}
 
-## CEL community libraries
-
-Kubernetes CEL expressions have access to the following CEL community libraries:
-
-- CEL standard functions, defined in the [list of standard definitions](https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions)
-- CEL standard [macros](https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros)
-- CEL [extended string function library](https://pkg.go.dev/github.com/google/cel-go/ext#Strings)
+## CEL options, language features and libraries
+
+CEL is configured with the following options, libraries and language features, introduced at the specified Kubernetes versions:
+
+| CEL option, library or language feature             | Included                                                                                                        | Availablity                |
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------- |
+| [standard macros](stdmacros)                        | `has`, `all`, `exists`, `exists_one`, `map`, `filter`                                                           | *                          |
+| [standard functions](stdlib)                        | see the [official list of standard definitions](stdlib)                                                         | *                          |
```

Comment:
nit: Does it make sense to capitalize Standard and See?

Reply:
What does the asterisk (`*`) indicate? All Kubernetes versions?
You may want to add a note/key about the use of that symbol.

Reply:
Good catch.  I was lazily using `*` as coder-speak for "wildcard".  I'll say "All Kubernetes releases" instead.